src/index.test.ts

```diff
@@ -0,0 +1,209 @@
+import * as path from 'path';
+import * as sinon from 'sinon';
+import { assert } from 'chai';
+import * as fontFinder from 'font-finder';
+
+import * as ligatureSupport from '.';
+
+describe('xterm-ligature-support', () => {
+  let onRefresh: sinon.SinonStub;
+  let term: MockTerminal;
+
+  // -> forms a ligature in Fira Code and Iosevka, but www only forms a ligature
+  // in Fira Code
+  const input = 'a -> b www c';
+
+  before(() => {
+    sinon.stub(fontFinder, 'list').returns(Promise.resolve({
+      'Fira Code': [{
+        path: path.join(__dirname, '../fonts/firaCode.otf'),
```

Comment:
There might be some license issue with including these in the repo, it might also be a good idea to download them as part of npm install?

Reply:
Both are licensed OFL, which is the MIT of the font world. I'll add a note about the fonts with links to their licenses at the bottom of the readme. Fetching them in a postinstall step is a good idea either way, so I'll add that too.